File: Level 0/MenorMayor/menormayor.cpp
#include <iostream>
using namespace std;
int main(){
    int a = 5, b = 1, c = 3;

    if (a > b && a > c) {
        cout << "A es el mayor";
        if (b < c)
            cout << "B es el menor";
        else
            cout << "C es el menor";
    } else if (b > a && b > c) {
        cout << "B es el mayor";
        cout << "A es el menor";
    } else {
        cout << "C es el mayor";
        cout << "A es el menor";
    }
    return 0;
}

File: Level 0/MenorMayor/menormayor.py
a = 1# int(input())
b = 2# int(input())
c = 3# int(input())

print(a < b < c)
print(a > b > c)

if a > b and a > c:
    print('A es el mayor')
    if b < c:
        print('B es el menor')
    else:
        print('C es el menor')
elif b > a and b > c:
    print('B es el mayor')
    print('A es el menor')
else:
    print('C es el mayor')
    print('A es el menor')

nums = [a, b, c]
min = nums[0]
max = nums[0]
for n in nums:
    if n < min:
        min = n
    if n > max:
        max = n

print(min, max)
